plugin/src/main/java/net/dzikoysk/funnyguilds/config/NumberRange.java

```diff
@@ -10,44 +10,48 @@
 import org.apache.commons.lang3.StringUtils;
 import panda.std.Option;
 
-public final class IntegerRange {
+public final class NumberRange {
```

Comment:
Create AbstractRange and implementations like NumberRange, IntegerRange, DoubleRange etc.